Paper: Developing a Graph Convolution-Based Analysis Pipeline for Multi-Modal Neuroimage Data: An Application to Parkinson's Disease

xtianmcd opened the pull request:
If you are creating this PR in order to submit a draft of your paper,
see http://procbuild.scipy.org/ for logs generated by the build
process.

See the [project readme](https://github.com/scipy-conference/scipy_proceedings#build-your-paper)
for more information.

xtianmcd committed:
working copy of first draft paper

xtianmcd committed:
Rename papers/neuro_analy_pipeline_scipy2019.rst to papers/christian_mcdaniel/neuro_analy_pipeline_scipy2019.rst

xtianmcd committed:
figures 1 and 3

xtianmcd committed:
resize fig 3

xtianmcd committed:
Update neuro_analy_pipeline_scipy2019.rst

xtianmcd committed:
sizing is incorrect

xtianmcd committed:
resize fig 3

xtianmcd committed:
Update neuro_analy_pipeline_scipy2019.rst

xtianmcd committed:
initial draft

deniederhut commented:
Hm... looks like you might have a bibtex issue. Here's the message from the server:

```
b'This is BibTeX, Version 0.99d (TeX Live 2017/Debian)\\nThe top-level auxiliary file: paper.aux\\nThe style file: alphaurl.bst\\nDatabase file #1: bibli.bib\\nWarning--I didn\\'t find a database entry for \"ZWAHYMKGC2013\"\\nWarning--to sort, need editor, organization, or key in ZHCLZW2018\\nWarning--to sort, need editor, organization, or key in KW2017\\nWarning--to sort, need editor, organization, or key in KZS2015\\nWarning--to sort, need editor, organization, or key in VCCRLB2018\\nWarning--to sort, need author or key in LZCY2017\\nWarning--to sort, need editor, organization, or key in CLCASSMBR1999\\nWarning--can\\'t use both volume and number fields in CLCASSMBR1999\\nWarning--empty school in K2018\\nWarning--empty publisher in Wang2015\\n(There were 10 warnings)\\nThis is pdfTeX, Version 3.14159265-2.6-1.40.18 (TeX Live 2017/Debian) (preloaded format=pdflatex)\\n restricted \\\\write18 enabled.\\nentering extended mode\\n(./paper.tex\\nLaTeX2e <2017-04-15>\\nBabel <3.12> and hyphenation patterns for 3 language(s) loaded.\\n(./IEEEtran.cls\\nDocument Class: IEEEtran 2015/08/26 V1.8b by Michael Shell\\n-- See the \"IEEEtran_HOWTO\" manual for usage information.\\n-- http://www.michaelshell.org/tex/ieeetran/\\n(/usr/share/texlive/texmf-dist/tex/latex/psnfss/ot1ppl.fd)\\n-- Using IEEE Computer Society mode.\\n-- Using 8.5in x 11in (letter) paper.\\n-- Using PDF output.\\n-- This is a 10 point document.\\n) (/usr/share/texlive/texmf-dist/tex/latex/cmap/cmap.sty)\\n(/usr/share/texlive/texmf-dist/tex/latex/base/ifthen.sty)\\n(/usr/share/texlive/texmf-dist/tex/latex/base/fontenc.sty\\n(/usr/share/texlive/texmf-dist/tex/latex/base/t1enc.def)\\n(/usr/share/texlive/texmf-dist/tex/latex/psnfss/t1ppl.fd)<<t1.cmap>>)\\n(/usr/share/texlive/texmf-dist/tex/latex/base/inputenc.sty\\n(/usr/share/texlive/texmf-dist/tex/latex/base/utf8.def\\n(/usr/share/texlive/texmf-dist/tex/latex/base/t1enc.dfu)\\n(/usr/share/texlive/texmf-dist/tex/latex/base/ot1enc.dfu)\\n(/usr/share/texlive/texmf-dist/tex/latex/base/omsenc.dfu)))\\n(/usr/share/texlive/texmf-dist/tex/latex/amsmath/amsmath.sty\\nFor additional information on amsmath, use the `?\\' option.\\n(/usr/share/texlive/texmf-dist/tex/latex/amsmath/amstext.sty\\n(/usr/share/texlive/texmf-dist/tex/latex/amsmath/amsgen.sty))\\n(/usr/share/texlive/texmf-dist/tex/latex/amsmath/amsbsy.sty)\\n(/usr/share/texlive/texmf-dist/tex/latex/amsmath/amsopn.sty))\\n(/usr/share/texlive/texmf-dist/tex/latex/graphics/color.sty\\n(/usr/share/texlive/texmf-dist/tex/latex/graphics-cfg/color.cfg)\\n(/usr/share/texlive/texmf-dist/tex/latex/graphics-def/pdftex.def))\\n(/usr/share/texlive/texmf-dist/tex/latex/caption/caption.sty\\n(/usr/share/texlive/texmf-dist/tex/latex/caption/caption3.sty\\n(/usr/share/texlive/texmf-dist/tex/latex/graphics/keyval.sty))\\n\\nPackage caption Warning: Unsupported document class (or package) detected,\\n(caption)                usage of the caption package is not recommended.\\nSee the caption package documentation for explanation.\\n\\n) (/usr/share/texlive/texmf-dist/tex/latex/float/float.sty) (./scipy.sty\\n(./status.sty (./draftwatermark.sty (./everypage.sty)\\n(/usr/share/texlive/texmf-dist/tex/latex/graphics/graphicx.sty\\n(/usr/share/texlive/texmf-dist/tex/latex/graphics/graphics.sty\\n(/usr/share/texlive/texmf-dist/tex/latex/graphics/trig.sty)\\n(/usr/share/texlive/texmf-dist/tex/latex/graphics-cfg/graphics.cfg)))))\\n(/usr/share/texlive/texmf-dist/tex/latex/psnfss/mathptmx.sty)\\n(/usr/share/texlive/texmf-dist/tex/latex/psnfss/helvet.sty)\\n(/usr/share/texlive/texmf-dist/tex/latex/psnfss/courier.sty)\\n(/usr/share/texlive/texmf-dist/tex/latex/amsfonts/amsfonts.sty)\\n(/usr/share/texlive/texmf-dist/tex/latex/amsfonts/amssymb.sty)\\n(/usr/share/texlive/texmf-dist/tex/latex/tools/bm.sty)\\n(/usr/share/texlive/texmf-dist/tex/latex/hyperref/hyperref.sty\\n(/usr/share/texlive/texmf-dist/tex/generic/oberdiek/hobsub-hyperref.sty\\n(/usr/share/texlive/texmf-dist/tex/generic/oberdiek/hobsub-generic.sty))\\n(/usr/share/texlive/texmf-dist/tex/generic/ifxetex/ifxetex.sty)\\n(/usr/share/texlive/texmf-dist/tex/latex/oberdiek/auxhook.sty)\\n(/usr/share/texlive/texmf-dist/tex/latex/oberdiek/kvoptions.sty)\\n(/usr/share/texlive/texmf-dist/tex/latex/hyperref/pd1enc.def)\\n(/usr/share/texlive/texmf-dist/tex/latex/latexconfig/hyperref.cfg)\\n(/usr/share/texlive/texmf-dist/tex/latex/url/url.sty))\\n\\nPackage hyperref Message: Driver (autodetected): hpdftex.\\n\\n(/usr/share/texlive/texmf-dist/tex/latex/hyperref/hpdftex.def\\n(/usr/share/texlive/texmf-dist/tex/latex/oberdiek/rerunfilecheck.sty))\\n\\nPackage hyperref Warning: Option `pdftex\\' has already been used,\\n(hyperref)                setting the option has no effect on input line 43.\\n\\n(/usr/share/texlive/texmf-dist/tex/latex/tools/longtable.sty)\\n(/usr/share/texlive/texmf-dist/tex/latex/multirow/multirow.sty)\\n(/usr/share/texlive/texmf-dist/tex/latex/base/textcomp.sty\\n(/usr/share/texlive/texmf-dist/tex/latex/base/ts1enc.def\\n(/usr/share/texlive/texmf-dist/tex/latex/base/ts1enc.dfu)))\\n(/usr/share/texlive/texmf-dist/tex/latex/upquote/upquote.sty)\\n(/usr/share/texlive/texmf-dist/tex/latex/fancyvrb/fancyvrb.sty\\nStyle option: `fancyvrb\\' v2.7a, with DG/SPQR fixes, and firstline=lastline fix \\n<2008/02/07> (tvz))) (./paper.aux)\\n(/usr/share/texlive/texmf-dist/tex/latex/base/ts1cmr.fd)\\n(/usr/share/texlive/texmf-dist/tex/latex/psnfss/t1ptm.fd)\\n-- Lines per column: 61 (exact).\\n(/usr/share/texlive/texmf-dist/tex/context/base/mkii/supp-pdf.mkii\\n[Loading MPS to PDF converter (version 2006.09.02).]\\n) (/usr/share/texlive/texmf-dist/tex/latex/oberdiek/epstopdf-base.sty\\n(/usr/share/texlive/texmf-dist/tex/latex/oberdiek/grfext.sty)\\n(/usr/share/texlive/texmf-dist/tex/latex/latexconfig/epstopdf-sys.cfg))\\n(/usr/share/texlive/texmf-dist/tex/latex/caption/ltcaption.sty)\\n(/usr/share/texlive/texmf-dist/tex/latex/hyperref/nameref.sty\\n(/usr/share/texlive/texmf-dist/tex/generic/oberdiek/gettitlestring.sty))\\n(./paper.out) (./paper.out)\\n(/usr/share/texlive/texmf-dist/tex/latex/psnfss/t1phv.fd)\\n(/usr/share/texlive/texmf-dist/tex/latex/psnfss/ot1ztmcm.fd)<<ot1.cmap>>\\n(/usr/share/texlive/texmf-dist/tex/latex/psnfss/omlztmcm.fd)<<oml.cmap>>\\n(/usr/share/texlive/texmf-dist/tex/latex/psnfss/omsztmcm.fd)<<oms.cmap>>\\n(/usr/share/texlive/texmf-dist/tex/latex/psnfss/omxztmcm.fd)<<omx.cmap>>\\n(/usr/share/texlive/texmf-dist/tex/latex/psnfss/ot1ptm.fd)\\n(/usr/share/texlive/texmf-dist/tex/latex/psnfss/upzd.fd)\\n(/usr/share/texlive/texmf-dist/tex/latex/psnfss/ts1ptm.fd) (./page_numbers.tex)\\n\\nLaTeX Warning: Citation `RST2014\\' on page 1 undefined on input line 209.\\n\\n\\nLaTeX Warning: Citation `GGLVVZ2018\\' on page 1 undefined on input line 209.\\n\\n\\nLaTeX Warning: Citation `BDH2016\\' on page 1 undefined on input line 209.\\n\\n\\nLaTeX Warning: Citation `LMSACRMW2018\\' on page 1 undefined on input line 209.\\n\\n\\nLaTeX Warning: Citation `LWXGXKZ2012\\' on page 1 undefined on input line 209.\\n\\n\\nLaTeX Warning: Citation `GGLVVZ2018\\' on page 1 undefined on input line 211.\\n\\n\\nLaTeX Warning: Citation `MLLAMWGSECES2018\\' on page 1 undefined on input line 21\\n1.\\n\\n\\nLaTeX Warning: Citation `BP2014\\' on page 1 undefined on input line 211.\\n\\n\\nLaTeX Warning: Citation `Brodal2016\\' on page 1 undefined on input line 211.\\n\\n\\nLaTeX Warning: Citation `ZYHJZLWWZZLLH2018\\' on page 1 undefined on input line 2\\n11.\\n\\n\\nLaTeX Warning: Citation `MLLAMWGSECES2018\\' on page 1 undefined on input line 21\\n1.\\n\\n\\nLaTeX Warning: Citation `GLHSA2014\\' on page 1 undefined on input line 211.\\n\\n\\nLaTeX Warning: Citation `TBEDTEEE2015\\' on page 1 undefined on input line 211.\\n\\n\\nLaTeX Warning: Citation `LSCZCCYLHGS2014\\' on page 1 undefined on input line 211\\n.\\n\\n\\nLaTeX Warning: Citation `GRSKMFZJHM2016\\' on page 1 undefined on input line 211.\\n\\n\\n\\nLaTeX Warning: Citation `BWSWBKSDRH2011\\' on page 1 undefined on input line 211.\\n\\n\\n\\nLaTeX Warning: Citation `TBEDTEEE2015\\' on page 1 undefined on input line 211.\\n\\n\\nLaTeX Warning: Citation `GRSKMFZJHM2016\\' on page 1 undefined on input line 211.\\n\\n\\n\\nLaTeX Warning: Citation `ZYHJZLWWZZLLH2018\\' on page 1 undefined on input line 2\\n13.\\n\\n\\nLaTeX Warning: Citation `MLLAMWGSECES2018\\' on page 1 undefined on input line 21\\n3.\\n\\n\\nLaTeX Warning: Citation `K2018\\' on page 1 undefined on input line 213.\\n\\n\\nLaTeX Warning: Citation `GLHSA2014\\' on page 1 undefined on input line 213.\\n\\n\\nLaTeX Warning: Citation `TBEDTEEE2015\\' on page 1 undefined on input line 213.\\n\\n\\nLaTeX Warning: Citation `CJMRCMBD2017\\' on page 1 undefined on input line 213.\\n\\n\\nLaTeX Warning: Citation `GRSKMFZJHM2016\\' on page 1 undefined on input line 213.\\n\\n\\n\\nLaTeX Warning: Citation `LCLZFFPK2015\\' on page 1 undefined on input line 214.\\n\\n\\nLaTeX Warning: Citation `BDH2016\\' on page 1 undefined on input line 214.\\n\\n\\nLaTeX Warning: Citation `LWXGXKZ2012\\' on page 1 undefined on input line 214.\\n\\n\\nLaTeX Warning: Citation `KPFRLGR2017\\' on page 1 undefined on input line 216.\\n\\n\\nLaTeX Warning: Citation `KPFRLGR2018\\' on page 1 undefined on input line 216.\\n\\n\\nLaTeX Warning: Citation `ZHCLZW2018\\' on page 1 undefined on input line 216.\\n\\n\\nLaTeX Warning: Citation `ZWAHYMKGC2013\\' on page 1 undefined on input line 218.\\n\\n\\nLaTeX Warning: Citation `Brodal2016\\' on page 1 undefined on input line 218.\\n\\n[1{/var/lib/texmf/fonts/map/pdftex/updmap/pdftex.map}]\\n\\nLaTeX Warning: Citation `Wang2015\\' on page 2 undefined on input line 236.\\n\\n\\nLaTeX Warning: Citation `HBL2010\\' on page 2 undefined on input line 236.\\n\\n\\nLaTeX Warning: Citation `Wang2015\\' on page 2 undefined on input line 242.\\n\\n\\nLaTeX Warning: Citation `HBL2010\\' on page 2 undefined on input line 242.\\n\\n\\nLaTeX Warning: Citation `Wang2015\\' on page 2 undefined on input line 244.\\n\\n\\nLaTeX Warning: Citation `HBL2010\\' on page 2 undefined on input line 244.\\n\\n\\nLaTeX Warning: No positions in optional float specifier.\\n               Default added (so using `tbp\\') on input line 253.\\n\\n\\nLaTeX Warning: No positions in optional float specifier.\\n               Default added (so using `tbp\\') on input line 257.\\n\\n[2 <./adj_mtx_fig.png>]\\n\\nLaTeX Warning: Citation `SNFOV2013\\' on page 3 undefined on input line 268.\\n\\n\\nLaTeX Warning: Citation `SNFOV2013\\' on page 3 undefined on input line 268.\\n\\n\\nLaTeX Warning: Command \\\\L invalid in math mode on input line 270.\\n\\n\\nLaTeX Warning: Command \\\\L invalid in math mode on input line 270.\\n\\n\\nLaTeX Warning: Command \\\\L invalid in math mode on input line 271.\\n\\n\\nLaTeX Warning: Command \\\\L invalid in math mode on input line 274.\\n\\n\\nLaTeX Warning: Command \\\\L invalid in math mode on input line 276.\\n\\n\\nLaTeX Warning: Citation `KW2017\\' on page 3 undefined on input line 276.\\n\\n\\nLaTeX Warning: Citation `KZS2015\\' on page 3 undefined on input line 289.\\n\\n\\nLaTeX Warning: Citation `KPFRLGR2017\\' on page 3 undefined on input line 289.\\n\\n\\nLaTeX Warning: Citation `ZHCLZW2018\\' on page 3 undefined on input line 289.\\n\\n\\nLaTeX Warning: Citation `VCCRLB2018\\' on page 3 undefined on input line 296.\\n\\n\\nLaTeX Warning: Citation `VCCRLB2018\\' on page 3 undefined on input line 298.\\n\\n\\nLaTeX Warning: Command \\\\textquoteright invalid in math mode on input line 298.\\n\\n(/usr/share/texlive/texmf-dist/tex/latex/psnfss/t1pcr.fd)\\n\\nLaTeX Warning: Command \\\\textquoteright invalid in math mode on input line 311.\\n\\n\\nLaTeX Warning: Command \\\\textquoteright invalid in math mode on input line 311.\\n\\n\\nLaTeX Warning: Command \\\\textquoteright invalid in math mode on input line 311.\\n\\n[3 <./feats_fig.png>]\\n\\nLaTeX Warning: No positions in optional float specifier.\\n               Default added (so using `tbp\\') on input line 316.\\n\\n\\nLaTeX Warning: Citation `MLLAMWGSECES2018\\' on page 4 undefined on input line 32\\n7.\\n\\n\\nLaTeX Warning: Citation `TBEDTEEE2015\\' on page 4 undefined on input line 327.\\n\\n\\nLaTeX Warning: Citation `BWSWBKSDRH2011\\' on page 4 undefined on input line 327.\\n\\n\\n\\nLaTeX Warning: Citation `LSCZCCYLHGS2014\\' on page 4 undefined on input line 327\\n.\\n\\n\\nLaTeX Warning: Citation `CJMRCMBD2017\\' on page 4 undefined on input line 327.\\n\\n\\nLaTeX Warning: Citation `GLHSA2014\\' on page 4 undefined on input line 327.\\n\\n\\nLaTeX Warning: Citation `K2018\\' on page 4 undefined on input line 327.\\n\\n\\nLaTeX Warning: Citation `ZYHJZLWWZZLLH2018\\' on page 4 undefined on input line 3\\n27.\\n\\n\\nLaTeX Warning: Citation `GRSKMFZJHM2016\\' on page 4 undefined on input line 327.\\n\\n\\n\\nLaTeX Warning: Citation `KUHSMHBB2016\\' on page 4 undefined on input line 327.\\n\\n\\nLaTeX Warning: Citation `HDCLPC2018\\' on page 4 undefined on input line 327.\\n\\n\\nLaTeX Warning: Citation `HCSAAP2014\\' on page 4 undefined on input line 327.\\n\\n\\nLaTeX Warning: Citation `LZCY2017\\' on page 4 undefined on input line 327.\\n\\n\\nLaTeX Warning: Citation `BSSNSOV2018\\' on page 4 undefined on input line 327.\\n\\n\\nLaTeX Warning: Citation `LCLZFFPK2015\\' on page 4 undefined on input line 327.\\n\\n\\nLaTeX Warning: Citation `BDH2016\\' on page 4 undefined on input line 327.\\n\\n\\nLaTeX Warning: Citation `LMSACRMW2018\\' on page 4 undefined on input line 327.\\n\\n\\nLaTeX Warning: Citation `LWXGXKZ2012\\' on page 4 undefined on input line 327.\\n\\n\\nLaTeX Warning: Citation `KW2017\\' on page 4 undefined on input line 329.\\n\\n\\nLaTeX Warning: Citation `VCCRLB2018\\' on page 4 undefined on input line 329.\\n\\n\\nLaTeX Warning: Citation `SNFOV2013\\' on page 4 undefined on input line 329.\\n\\n\\nLaTeX Warning: Citation `KZS2015\\' on page 4 undefined on input line 329.\\n\\n\\nLaTeX Warning: Citation `KCR2016\\' on page 4 undefined on input line 329.\\n\\n\\nLaTeX Warning: Citation `ZHCLZW2018\\' on page 4 undefined on input line 329.\\n\\n\\nLaTeX Warning: Citation `KPFRLGR2017\\' on page 4 undefined on input line 329.\\n\\n\\nLaTeX Warning: Citation `KPFRLGR2018\\' on page 4 undefined on input line 329.\\n\\n\\nLaTeX Warning: Citation `ZHCLZW2018\\' on page 4 undefined on input line 329.\\n\\n[4 <./GCNetwork_fig.png>]\\n\\nLaTeX Warning: Citation `MCCZ1999\\' on page 5 undefined on input line 357.\\n\\n\\nLaTeX Warning: Citation `BPPDA2000\\' on page 5 undefined on input line 357.\\n\\n\\nLaTeX Warning: Citation `LWTJAM2003\\' on page 5 undefined on input line 357.\\n\\n\\nLaTeX Warning: Citation `CLCASSMBR1999\\' on page 5 undefined on input line 357.\\n\\n\\nLaTeX Warning: Citation `ZZWJJPNLYT2015\\' on page 5 undefined on input line 357.\\n\\n\\n\\nLaTeX Warning: Reference `featfig\\' on page 5 undefined on input line 368.\\n\\n\\nLaTeX Warning: Citation `KW2017\\' on page 5 undefined on input line 375.\\n\\n\\nLaTeX Warning: Citation `VCCRLB2018\\' on page 5 undefined on input line 377.\\n\\n[5]\\n\\nLaTeX Warning: Citation `SNFOV2013\\' on page 6 undefined on input line 417.\\n\\n\\nLaTeX Warning: Command \\\\L invalid in math mode on input line 419.\\n\\n\\nLaTeX Warning: Command \\\\L invalid in math mode on input line 419.\\n\\n\\nLaTeX Warning: Command \\\\L invalid in math mode on input line 420.\\n\\n\\nLaTeX Warning: Command \\\\L invalid in math mode on input line 421.\\n\\n\\nLaTeX Warning: Command \\\\L invalid in math mode on input line 423.\\n\\n\\nLaTeX Warning: Citation `SNFOV2013\\' on page 6 undefined on input line 423.\\n\\n\\nLaTeX Warning: Citation `M2009\\' on page 6 undefined on input line 423.\\n\\n\\nLaTeX Warning: Command \\\\L invalid in math mode on input line 423.\\n\\n\\nLaTeX Warning: Citation `HBL2015\\' on page 6 undefined on input line 423.\\n\\n\\nLaTeX Warning: Command \\\\L invalid in math mode on input line 427.\\n\\n\\nLaTeX Warning: Command \\\\L invalid in math mode on input line 427.\\n\\n\\nLaTeX Warning: Citation `KW2017\\' on page 6 undefined on input line 427.\\n\\n\\nLaTeX Warning: Command \\\\L invalid in math mode on input line 429.\\n\\n\\nLaTeX Warning: Citation `HVG2011\\' on page 6 undefined on input line 429.\\n\\n\\nLaTeX Warning: Command \\\\textquoteright invalid in math mode on input line 429.\\n\\n\\nLaTeX Warning: Command \\\\textquoteright invalid in math mode on input line 429.\\n\\n\\nLaTeX Warning: Command \\\\L invalid in math mode on input line 429.\\n\\n\\nLaTeX Warning: Command \\\\L invalid in math mode on input line 429.\\n\\n\\nLaTeX Warning: Command \\\\L invalid in math mode on input line 429.\\n\\n\\nLaTeX Warning: Command \\\\textquoteright invalid in math mode on input line 430.\\n\\n\\nLaTeX Warning: Command \\\\textquoteright invalid in math mode on input line 430.\\n\\n\\nLaTeX Warning: Command \\\\L invalid in math mode on input line 430.\\n\\n\\nLaTeX Warning: Command \\\\L invalid in math mode on input line 430.\\n\\n\\nLaTeX Warning: Citation `KW2017\\' on page 6 undefined on input line 431.\\n\\n\\nLaTeX Warning: Command \\\\L invalid in math mode on input line 433.\\n\\n\\nLaTeX Warning: Citation `KW2017\\' on page 6 undefined on input line 433.\\n\\n\\nLaTeX Warning: Citation `KW2017\\' on page 6 undefined on input line 435.\\n\\n\\nLaTeX Warning: Command \\\\textquoteright invalid in math mode on input line 435.\\n\\n\\nLaTeX Warning: Command \\\\textquoteright invalid in math mode on input line 435.\\n\\n\\nLaTeX Warning: Command \\\\textquoteright invalid in math mode on input line 435.\\n\\n\\nLaTeX Warning: Command \\\\L invalid in math mode on input line 435.\\n\\n\\nLaTeX Warning: Command \\\\textquoteright invalid in math mode on input line 435.\\n\\n\\nLaTeX Warning: Command \\\\textquoteright invalid in math mode on input line 435.\\n\\n\\nLaTeX Warning: Command \\\\textquoteright invalid in math mode on input line 435.\\n\\n\\nLaTeX Warning: Command \\\\textquoteright invalid in math mode on input line 435.\\n\\n\\nLaTeX Warning: Command \\\\textquoteright invalid in math mode on input line 435.\\n\\n\\nLaTeX Warning: Citation `KW2017\\' on page 6 undefined on input line 437.\\n\\n\\nPackage hyperref Warning: Ignoring empty anchor on input line 446.\\n\\n\\nPackage hyperref Warning: Ignoring empty anchor on input line 448.\\n\\n\\nPackage hyperref Warning: Ignoring empty anchor on input line 450.\\n\\n\\nPackage hyperref Warning: Ignoring empty anchor on input line 452.\\n\\n\\nPackage hyperref Warning: Ignoring empty anchor on input line 454.\\n\\n\\nPackage hyperref Warning: Ignoring empty anchor on input line 456.\\n\\n\\nPackage hyperref Warning: Ignoring empty anchor on input line 458.\\n\\n\\nPackage hyperref Warning: Ignoring empty anchor on input line 460.\\n\\n\\nPackage hyperref Warning: Ignoring empty anchor on input line 462.\\n\\n\\nPackage hyperref Warning: Ignoring empty anchor on input line 464.\\n\\n\\nPackage hyperref Warning: Ignoring empty anchor on input line 466.\\n\\n\\nPackage hyperref Warning: Ignoring empty anchor on input line 468.\\n\\n\\nPackage hyperref Warning: Ignoring empty anchor on input line 470.\\n\\n\\nPackage hyperref Warning: Ignoring empty anchor on input line 472.\\n\\n(./paper.bbl\\n\\n! Package inputenc Error: Unicode char \\xce\\xb1 (U+3B1)\\n(inputenc)                not set up for use with LaTeX.\\n\\nSee the inputenc package documentation for explanation.\\nType  H <return>  for immediate help.\\n ...                                              \\n                                                  \\nl.25 ...uroimaging and behavioral assessment of \\xce\\xb1\\n                                                  -synuclein\\n!  ==> Fatal error occurred, no output PDF file produced!\\nTranscript written on paper.log.\\n'<string>:150: (WARNING/2) Explicit markup ends without a blank line; unexpected unindent.\n
```

xtianmcd committed:
fixes re non-build

xtianmcd committed:
fixes re non-build

xtianmcd committed:
Merge branch '2019' of https://github.com/xtianmcd/scipy_proceedings into 2019

xtianmcd committed:
test build

xtianmcd committed:
fix build warning

xtianmcd committed:
fix footnote warning (attempt)

xtianmcd committed:
fix warning re footnotes (attempt)

xtianmcd committed:
attempt to debug bib file

xtianmcd committed:
fix re non-build

xtianmcd commented:
@deniederhut thanks for your comments. Where did you find that server output? My `.log` output and the output printed to Terminal wasn't showing the error regarding the alpha character in my `.bib` file. 

Anyway, I've fixed that error, and I figured out the `push` issue (needed to specify to push to `origin 2019`). However, I still cannot get a successful build. FYI, I have verified my `.bib` file using  the `BibLatex-Check` package (although I know BibLatex is technically different from a bibfile), I have tried running `make clean` before the build, and have tried repeating the build twice to get the `.bib` file to build (these were all suggestions from you last year when I encountered similar problems). I get a successful pdf locally (except for citations), and I don't see any errors in my `.log` file (I can provide it if that helps), but I do see warnings. 

I didn't expect the warnings to prevent a build (else I figured they would be errors), but I'm trying to fix them now. I've resolved all but a few Warning *types* that I don't know how to resolve. E.g.:
 
```
Package hyperref Warning: Ignoring empty anchor on input line 419
``` 
I've checked my footnotes and they are all entered according to the [docs](http://docutils.sourceforge.net/docs/user/rst/quickref.html#footnotes). 

```
LaTeX Warning: Citation `KW2017' on page 6 undefined on input line 406.
``` 
The citation is in the `.bib` file as 
```
@Proceedings{KW2017,
  author = "Thomas N Kipf and Max Welling",
  title = "Semi-Supervised Classification with Graph Convolutional Networks",
  year = "2017",
  publisher = "ICLR",
}
```
and the `.bib` file is noted in my `.rst` file using `:bibliography: <bibfile_name_without_extension>` at the top. There is a Warning like this for most or all of my citations, indicating there's still an issue in my `.bib` file. Finding the location you got that other error from may be what I need.  

```
Underfull \hbox (badness 3343) in paragraph at lines ... 
```
and 
```
Package caption Warning: Unsupported document class (or package) detected,
(caption)                usage of the caption package is not recommended.
See the caption package documentation for explanation.
```
seem related to the translation between `.rst` and Latex, and I couldn't find fixes for them from my end (in my `.rst` file). 

Anyway, these are my thoughts... pls let me know if you have any other suggestions, and where you got the output from the server that you posted earlier. Thanks so much!! I'm learning a lot

xtianmcd committed:
added raw latex re bibfile

xtianmcd committed:
escape hyphen in bibfile

xtianmcd committed:
remove hyphen in bibfile

cbcunc commented:
That last commit did it. It builds on procbuild now.

xtianmcd committed:
attempt to move reference above appendix A

xtianmcd commented:
Success! Thanks for the help and patience. FYI, I got my references to appear by adding the lines 
```
.. raw:: latex

   \bibliographystyle{plain}
```
to my `.rst` file. Initial answer [here](https://stackoverflow.com/a/15296591) although I did not need the lines from the first code block or the bottom line of the last block to make the References appear (likely because of the `:bibliography: <bibfile_name_without_extenstion>` line already in the `.rst` file).

I added the lines before my Appendix section to try and put the References before the Appendix, but the References still inserted at the end. Any idea how to "append" an Appendix to the very end, after References?. 

These lines also caused helpful errors to appear in my `.log` file and print to Terminal (I had a problematic hyphen in my `.bib` file). I tried building without the raw latex lines after fixing the hyphen and the References did not show up, so I confirmed that these lines are the reason for the References appearing in my case.

deniederhut commented:
I stopped by to look at the error logs but it looks like you've already got it figured out - congrats!

AngelosPsy commented:
Dear @xtianmcd,

I am one of the reviewers of your paper. Would you prefer me to list my comments in a comment box or on the [rst](https://github.com/scipy-conference/scipy_proceedings/blob/b150fad0ec676c7a46dab24bdc5f60246b902a74/papers/christian_mcdaniel/neuro_analy_pipeline_scipy2019.rst) file?

xtianmcd commented:
Hi @AngelosPsy, thanks for checking in! I would say, if you have some comments that point to specific words/sentences, then on the rst file will be best. Otherwise, more general comments about a section, etc may be better to leave here To keep more of a record. 

Also FYI, I will be adding more to the results later this week. I have the results but am currently out of town. Thanks!

AngelosPsy commented:
Hi @xtianmcd,

OK, since from what I understand the paper is not 100% ready yet (i.e., the results are missing) I will just place my general comments here. After revisions, I can go through the paper sentence by sentence and make more specific comments. I understand that you are out of town so I do not expect and immediate answer. Also, if you are on vacation, please feel free to ignore my message until you return – it is good to have real vacation :smiley:

_Copyright_
-	Why not just write “Christian McDaniel,  Shannon Quinn” rather than “Christian McDaniel et al.” ?

_Abstract_
-	“resulting in progressively worsening motor, cognitive, and psychological symptoms.” What do you mean with “worsening…psychological symptoms”?
-	The abstract may turn out to be too long after you insert the results. Maybe you should rewrite the abstract a bit so it is not too long.

_Introduction_
-	Maybe when there are multiple references you can include them within brackets.
-	In general, I think the introduction assumes too much knowledge from the readers. For example, I do not think many would know what “T1-weighted images” are. The same goes for some statements such as the following: “The edges of the graph offer an explicit metric for establishing meaningful neighborhoods of nodes and the features offer meaningful descriptors with which to group them.” It should be clear what it is meant with “meaningful descriptors” etc. 
-	Please also check throughout the manuscript and remove some bits that seem like residual notes (e.g., “[and functional data??]” or “[If functional data is used, insert info re: fMRI acquisition/preprocessing]”).

_Background_
-	I absolutely understand why there is such extensive explanation of, for example, the Graph Convolutional Networks etc. However, I soon got lost in that section and I think the way this part is written targets more of a specialized audience rather than the broader audience of scipy. I would kindly suggest that big parts of the background section are reduced so that the paper focuses more on the programming part (see Methods section). To repeat, I absolutely agree that a good background will make it easier to understand the methods but I do think that the background part is written for more of a general, rather than a specialized, audience.

_Discussions and Conclusions_
-	This part needs to be rewritten as it is neither a discussion or a conclusion section. It is more of a one-line summary of the paper. 

_References_
-	There is a mix up of the footones with the references. I would suggest that the footnoes are placed before the references.

xtianmcd committed:
reorganization and revision re: background/ related works/ methods sections

xtianmcd committed:
correction of issue in in-text math (\textbf{W_{a}} needed to be \textbf{W}_{a})), which had not been an issue previously...

xtianmcd committed:
revisions, results, discussion, conclusion

xtianmcd committed:
moved footnotes

xtianmcd commented:
Hi @AngelosPsy,

Thanks for your helpful feedback, and patience regarding my travels! I have addressed your points, and have made some comments below. 

In general, I reduced the Background and combined it with the Methods section, and reworded/revised a lot of the paper. My goal is to include enough technical background for neuroimaging and deep learning so that an outsider to the fields could understand and implement the methods, in line with the "pipeline" format. However, that's easier said than done and if things are still too wordy or verbose please let me know! 

_Copyright_
> Why not just write “Christian McDaniel, Shannon Quinn” rather than “Christian McDaniel et al.” ?

This is generated by the build system of SciPy. Do you know if there is a way for me to change it? If so, I'll look into it!

_Abstract_
> “resulting in progressively worsening motor, cognitive, and psychological symptoms.” What do you mean with “worsening…psychological symptoms”?

I changed this to 'psychiatric symptoms (i.e., depression)'. I am referring to issues such as depression. Hopefully that makes it more clear; otherwise I can try something else.

> The abstract may turn out to be too long after you insert the results. Maybe you should rewrite the abstract a bit so it is not too long.

Do you know the limit? I believe newly updated abstract is currently exactly 150 words. 


> There is a mix up of the footones with the references. I would suggest that the footnoes are placed before the references.

This is also a result of the build process. I tried moving the footnotes but they were still inserted in the same place. (I have also tried putting the Appendices after the Results, but the build process seems to always put the Results at the very end)

xtianmcd committed:
removed erroneous sentence

xtianmcd committed:
change AUC scores from % to decimal; correct name of kernel used in SVM, fix line breaks

AngelosPsy commented:
Thank you for your responses to my answers. Could you maybe fix the file as it fails to build (see here http://procbuild.scipy.org/# ). After that I can continue with my review.

xtianmcd committed:
fixed malformed table

xtianmcd commented:
Thanks @AngelosPsy for catching that... Try now!

cbcunc reviewed (changes requested):
@xtianmcd This paper renders to 10 pages. Our review guidelines state, "The compiled version should be no longer than 8 pages, including figures, source code, appendices, etc.."

AngelosPsy commented:
Hi @xtianmcd,

I do not know how the text would look like after you have reduced it to fit in the 8-page limit but here are some additional comment to the present version of the ms.


-	“Parkinson’s disease (PD) is a highly prevalent neurodegenerative disease” : Disease is a … disease. Please rephrase.
-	Figure 4: Increase label size, do you need colors? Is not the the importance obvious via the length of the bar. You may also consider decreasing the width of the bars to save space.

-	Please fix the gap on page 8

-	“MRI data requires extensive artifact correction and removal before it can be used. MRI signals are acquired through the application of precisely coordinated magnetic fields and radiofrequency (RF) pulses.”: Please combine sentences

-	The standard software for preprocessing anatomical MRI (aMRI1) data is Freesurfer9. Freesurfer is an actively developed software with responsive technical support and rich forums. The software is dense and the documentation is lacking in some areas, so training may still be helpful, although not available in our case.: Please reduce redundancy here so that the text is shortened.

-	Discussions and Conclusions: I think it should be ‘Discussion and Conclusion’

xtianmcd committed:
made suggested revisions

xtianmcd commented:
@AngelosPsy I have made the suggested changes. 

@cbcunc Is it okay if the Footnotes and References exceed the 8-page limit? If so, then the paper is now 8 pages, except that it insists on keeping the last line on page 9. When I shorten sections, the build process just widens that gaps of the math sections to keep the last line on page 9. Any advice?

I am not seeing the changes in the build server. Do you I need to create a pull request, or was that done when I pushed to my fork and the changes are just waiting to be approved?

deniederhut commented:
> I am not seeing the changes in the build server.

Builds can be triggered by hand by pressing the refresh button on the right side of your PR listing on procbuild.scipy.org

AngelosPsy commented:
Hi @xtianmcd,

I am OK with the paper -- the pdf build works http://procbuild.scipy.org/download/xtianmcd-2019. However, I am going to give my OK until it is clear whether it is OK to submit a paper that is longer than the 8 pages -- I do not think so as it is against the guidelines as mentioned by @cbcunc. If it is not, then I will go over the revised paper -- this means the shorter version after you have revised the paper -- before giving my green light. BTW, we are past the deadline...

deniederhut commented:
So this still looks like it's building at 10 pages. I certainly wouldn't want to dissuade you from being generous with your citations, so maybe instead we might think about removing the appendices? It might be enough to provide a brief description of dwi (it is some metric of water movement) with a reference to a suggested textbook if people want to know more about t1 decay. We could probably do something similar with your description of graph convolutional networks.

xtianmcd committed:
removed appendices

xtianmcd commented:
Sorry @deniederhut , I had thought that the References could exceed the 8pg limit. I have removed the appendices, but the build is just adding excess whitespace in earlier sections and keeping part of the references on p.9. If the extra whitespace was removed, it would now fit on 8 pages. 

Also, I don't see my most recent commit in the build server, even after refreshing the PR. I think it may need manual intervention since it says "Changes requested" under my latest commit...

xtianmcd committed:
minor change

deniederhut commented:
> I don't see my most recent commit in the build server

Paper refreshing is limited to once every 5 minutes. Maybe you are running in to this? I see your latest version just fine.

<img width="375" alt="Screenshot 2019-06-30 15 46 38" src="https://user-images.githubusercontent.com/6868396/60401940-53f79680-9b4e-11e9-83d0-5ef9d9930821.png">

Oh crap yeah this is super gross. So latex is mad because you piled up all of your footnotes at the bottom of the file. If you distribute them where they occur in the text (you can see the 00_vanderwalt example), this will typeset without giant blocks of empty space. 

Two more things:
1. If you need to reference a footnote several times throughout the paper, it might want to be a citation instead
2. Please don't modify the document style (line 237)

xtianmcd committed:
distributed footnotes

xtianmcd commented:
@deniederhut thanks for the feedback. I distributed the footnotes and the paper is now coming in under 8 pages!

Regarding line 237. That was the only way I could figure out to get the References to show up. When I remove that line, the References section disappears. Do you have any advice to get the References to show up without the raw latex lines?

deniederhut commented:
When I remove that line, the references section is still there. Maybe something else was happening?

xtianmcd committed:
removed raw latex line

xtianmcd commented:
Ah, the issue was that without the raw latex lines, the local build was did not include the References, while the build on the build server does.

xtianmcd committed:
shortened GCN background section to accommodate longer References section

xtianmcd committed:
smaller image to reduce paper length

cbcunc reviewed (approved):
@xtianmcd Thanks for fixing the paper length. That was the last hang-up and now the paper is approved.

deniederhut commented:
Woohoo! Thanks for your hard work everyone!

xtianmcd commented:
Hi @deniederhut, I realize the extreme lateness here, but the other author on the paper has asked to be added as a corresponding author. His email is spq@uga.edu. I understand if you cannot make this change so late, but I wanted to check if it might be possible! Thanks for everything.

deniederhut commented:
Sure thing - if you open a PR with the change in the couple of days we can get this in time for the finalized proceedings.

xtianmcd commented:
Thanks very much. Do you happen to know how to indicate the corresponding author? I can't seem to find how to do it in the document and I think I remember just listing the corresponding authors on a form when I submitted my abstract.. The other author's email address is included in the `.rst` file.

deniederhut commented:
Yeah, there is a `:corresponding:` tag you can put underneath the corresponding author. This should be visible in the `00_vanderwalt` example.

xtianmcd commented:
Thank you. I just pushed the update. I don't see it in the build server though, do you know if my changes went through?
